build/LegacyPackages/14.0/Microsoft.Build.Framework/Microsoft.Build.Framework.nuspec

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
+  <metadata minClientVersion="2.8.1">
```

Comment:
This is a signal to NuGet that only version 2.8.1 and above will be able to fully understand it.  This is because elements like <group /> under a <dependencies /> element were introduced in 2.8.1.

---
In reply to: [75878987](https://github.com/Microsoft/msbuild/pull/929#discussion_r75878987) [](ancestors = 75878987)